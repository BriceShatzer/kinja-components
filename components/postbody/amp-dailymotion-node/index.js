// @flow

import AmpDailyMotionNode from './amp-dailymotion-node';

export default AmpDailyMotionNode;
