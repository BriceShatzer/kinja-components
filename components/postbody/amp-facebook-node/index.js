// @flow

import AmpFacebookNode from './amp-facebook-node';
import AmpFacebookPostNode from './amp-facebookPost-node';

export {
	AmpFacebookNode,
	AmpFacebookPostNode
};