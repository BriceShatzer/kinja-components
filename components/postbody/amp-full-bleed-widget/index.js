// @flow

import AmpFullBleedWidget from './amp-full-bleed-widget';

export default AmpFullBleedWidget;
