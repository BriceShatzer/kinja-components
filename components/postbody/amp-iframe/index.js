// @flow

import AmpIframe from './amp-iframe';

export default AmpIframe;
