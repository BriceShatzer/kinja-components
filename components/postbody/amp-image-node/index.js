// @flow

import AmpImageNode from './amp-image-node';

export default AmpImageNode;
