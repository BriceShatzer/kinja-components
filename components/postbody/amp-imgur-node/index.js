// @flow

import AmpImgurNode from './amp-imgur-node';

export default AmpImgurNode;
