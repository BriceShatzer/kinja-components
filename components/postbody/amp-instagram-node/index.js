// @flow

import AmpInstagramNode from './amp-instagram-node';

export default AmpInstagramNode;
