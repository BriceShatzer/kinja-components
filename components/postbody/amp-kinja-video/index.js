import AmpKinjaVideo from './amp-kinja-video';

export default AmpKinjaVideo;
