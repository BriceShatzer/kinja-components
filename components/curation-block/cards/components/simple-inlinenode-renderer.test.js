import simpleInlineNodes from './simple-inlinenode-renderer';
import { TextNode, LinkNode, LineBreakNode } from 'postbody/InlineNode';

describe('<InlineNode />', () => {

	describe('should render text', () => {
		it('should render italic', () => {
			const result = simpleInlineNodes([new TextNode('Kinja', ['Italic'])]);
			expect(result).toMatchSnapshot();
		});
		it('should ignore links, linebreaks and other styles', () => {
			const result = simpleInlineNodes([
				new TextNode('Kinja', ['Italic', 'Underline']),
				new LineBreakNode(),
				new LinkNode([new TextNode('This is link')], 'href')
			]);
			expect(result).toMatchSnapshot();
		});
	});
});
