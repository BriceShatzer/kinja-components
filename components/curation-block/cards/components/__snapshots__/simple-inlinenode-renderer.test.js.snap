// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InlineNode /> should render text should ignore links, linebreaks and other styles 1`] = `"<em>Kinja</em>This is link"`;

exports[`<InlineNode /> should render text should render italic 1`] = `"<em>Kinja</em>"`;
